lib/resources/r4/related_person.yaml

```diff
@@ -229,7 +229,7 @@ fields:
   action: create
   note: |
     <ul>
-      <li>If provided, the <code>valueCodeableConcept.coding[0].code</code> must be `Encounter`.</li>
+      <li>The <code>valueCodeableConcept.coding[0].code</code> can be `Encounter or Patient`.</li>
```

Comment:
space after <li> and space before </li> applies through out the file

Reply:
updated: https://github.com/cerner/fhir.cerner.com/pull/689/commits/6ef20fc0201eeb01007296ad539296c4bcf3250e#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R232